cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2442,7 +2436,56 @@ Slice::Python::CodeVisitor::writeDocstring(const EnumPtr& p, Output& out)
 }
 
 void
-Slice::Python::CodeVisitor::writeDocstring(const OperationPtr& op, MethodKind methodKind, Output& out)
+Slice::Python::CodeVisitor::writeConstDocstring(const ConstPtr& p, IceInternal::Output& out)
+{
+    const optional<DocComment>& comment = p->docComment();
```

Comment:
Do both pylance (whatever vscode uses) and sphinx play nice with the `#:` format?